src/transformers/onnx/features.py

```diff
@@ -209,6 +210,16 @@ class FeaturesManager:
             "token-classification",
             onnx_config_cls=LayoutLMOnnxConfig,
         ),
+        "electra": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "causal-lm",
+            "sequence-classification",
+            # "multiple-choice",
```

Comment:
Maybe we can delete this commented out code?

Reply:
I deleted except sequence-classification because i am using electra model for sequence-classification.
That's why i started this PR.
Thanks!